sdk/cmake/host-tools.cmake

```diff
@@ -118,3 +118,17 @@ function(setup_host_tools)
         add_dependencies(native-${_module} host-tools ${INSTALL_DIR}/bin/${HOST_EXTRA_DIR}${_module}${HOST_MODULE_SUFFIX})
     endforeach()
 endfunction()
+
+function(generate_whitelist_code _whitelist_file)
+
+    # Error out on anything else than txt
+    if(NOT ${_whitelist_file} MATCHES ".*\\.txt")
```

Comment:
is the match string a regex? or some windows-like expression? (looks strange at first sight, I would have expected something like `"*.txt"`)

Reply:
Look at the cmake documentation?

Reply:
In fact I think it needs another `` \``